.github/workflows/pr-labeler.yaml

```diff
@@ -0,0 +1,12 @@
+name: pr-labeler
+on:
+  - pull_request
```

Comment:
Have you confirmed whether it works with PRs from forks? They have different permissions.

Reply:
To understand the difference, please search by the word `pull_request_target`.

https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#pull_request_target

Reply:
Not yet and I didn't notice that.
(In the first place, I should have create PR from the fork)
So it should follow like the other PR e.x.  https://github.com/autowarefoundation/autoware.universe/blob/main/.github/workflows/pre-commit.yaml#L8-L16 ?

Reply:
`pre-commit.yaml` isn't related. These are the examples.
https://github.com/autowarefoundation/autoware.universe/blob/a2bcd7c531bffa5b7966f1c1b751a59733bfdcc6/.github/workflows/semantic-pull-request.yaml#L4
https://github.com/autowarefoundation/autoware.universe/blob/a2bcd7c531bffa5b7966f1c1b751a59733bfdcc6/.github/workflows/deploy-docs.yaml#L13

Reply:
I finally understand it, thank you.
Reflected it by https://github.com/autowarefoundation/autoware.universe/pull/2266/commits/1a13aa2e91c91c50793285fa943cda75fd678784